test/testrunner.py

```diff
@@ -189,7 +189,13 @@ def pull_dlc_images(images):
 
 
 def setup_eks_cluster(framework_name, is_neuron):
-    frameworks = {"tensorflow": "tf", "pytorch": "pt", "mxnet": "mx"}
+    frameworks = {
+        "tensorflow": "tf",
+        "mxnet": "mx",
+        "pytorch": "pt",
+        "huggingface_pytorch": "hf-pt",
+        "huggingface_tensorflow": "hf-tf",
```

Comment:
If we don't test HF DLCs on EKS, why do we need this?

Reply:
It's not good that we have the same map in 2 different places. But at least I was trying to keep it consistent.

Reply:
Makes sense. In that case, we could pull it out into a different function.